src/translations/strings.yaml

```
@@ -928,6 +928,25 @@ devices:
     value: My devices
     comment: Title for the menu bar when viewing the devices or device limit pages
 
+  excludedAppsTitle:
+    value: Excluded apps
+    comment: Title label for the excluded apps help sheet
+  excludedAppsHeader:
+    value: What are excluded apps? 
+    comment: Header label for the excluded apps help sheet
+  excludedAppsBody1:
+    value: Excluded apps let you turn off VPN protection for specific apps, without turning off your device’s VPN protection.
+    comment: Body label for the excluded apps help sheet
+  excludedAppsBody2:
+    value: When you exclude an app, your IP address and approximate location will be exposed to it.
```

```suggestion
    value: When you exclude an app, it will have access to your IP address and approximate location.
```

In case content needs to check these strings, this feels a bit easier to read.

---

@traceyvantyghem

---

Sure, I also like this suggestion. 

Thanks Flod.